pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[tool.black]
+line-length = 100
```

Comment:
I think it says `115` in `.flake8`. Normal?

Reply:
Not sure tbh. It might be needed to not error out, but I'll do some tests.